src/mudlet-lua/lua/geyser/GeyserMiniConsole.lua

```diff
@@ -26,6 +26,18 @@ function Geyser.MiniConsole:setBufferSize (linesLimit, sizeOfBatchDeletion)
   setConsoleBufferSize(self.name, linesLimit, sizeOfBatchDeletion)
 end
 
+--- Sets the new font to use - use a monospaced font, non-monospaced fonts aren't supported by Mudlet
```

Comment:
I would not bother to put that in place - I am ***actively working on changing that*** - because for some non-Latinate writing systems there are no *monospaced* fonts and Qt will automatically grab needed glyphs from proportional ones if it needs to... once I (or maybe @novload) gets something working to everyone's satisfaction in the next few days (week).

Reply:
You're actively working on a lot of things :wink: I will update the text once the code is in.